File: WebAPI/WebAPI/Model/sys_khoa_model.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebAPI.System;

namespace WebAPI.Model
{
    public class sys_khoa_model
    {
        public sys_khoa_model()
        {
            db = new sys_khoa();
        }
        public sys_khoa db { get; set; }
        public string create_name { get; set; }
        public string update_name { get; set; }
    }
    public class filter_data_khoa
    {
        public string search { get; set; }
        public string total { get; set; }
        public string page { get; set; }
        public string limit { get; set; }
        public string status_del { get; set; }

    }
}

File: WebAPI/ClientApp/src/app/model/sys_khoa.model.ts
import { sys_khoa } from "../database/sys_khoa.data";

export class sys_khoa_model {
  db: sys_khoa;
  create_name: string;
  update_name: string;
}
export class filter_data_khoa {
  search: string;
  total: number | null;
  page: number | null;
  limit: number | null;
}
